internal/js-dependency-scanner/src/main/java/org/wso2/security/tools/scanner/dependency/js/utils/ConfigParser.java

```diff
@@ -0,0 +1,248 @@
+                .toCharArray(), properties.getProperty(JSScannerConstants.PASSWORD).toCharArray(),
+                properties.getProperty(JSScannerConstants.SECURITY_ARTIFACT_REPO));
+        return reportUploader;
+
+    }
+
+    /**
+     * Parse github credential details (Access-token, Username, Password).
+     *
+     * @throws ConfigParserException exception occurred while parsing configuration details.
+     */
+    public static void parseGitAccessToken() throws ConfigParserException {
+
+        Properties properties = new Properties();
+        ClassLoader classLoader = ConfigParser.class.getClassLoader();
+        try (InputStream input = classLoader.getResourceAsStream(JSScannerConstants.GIT_ACCESS_TOKEN_CONFIG_FILE)) {
+            properties.load(input);
+        } catch (IOException e) {
+            throw new ConfigParserException("Error occurred in parsing github credentials : " + e);
+        }
+        CommonApiInvoker.setGitToken(properties.getProperty(JSScannerConstants.ACCESSTOKEN).toCharArray());
+    }
+
+    /**
+     * Parse jira credential details (Username, Password).
+     *
+     * @throws ConfigParserException exception occurred while parsing configuration details.
+     */
+    public static IssueCreator parseIssueTicketCreatorCredentials() throws ConfigParserException {
+
+        Properties properties = new Properties();
+        ClassLoader classLoader = ConfigParser.class.getClassLoader();
+        try (InputStream input = classLoader.getResourceAsStream(JSScannerConstants.ISSUECREATOR_CONFIG_FILE)) {
+            properties.load(input);
+        } catch (IOException e) {
+            throw new ConfigParserException("Error occurred in parsing JIRA Credentials : " + e);
+        }
+
+        IssueCreator issueCreatorAPI = new JIRAIssueCreator(new JIRARestClient(), properties.getProperty
+                (JSScannerConstants.USERNAME).toCharArray(),
+                properties.getProperty(JSScannerConstants.PASSWORD).toCharArray(),
+                properties.getProperty(IssueCreatorConstants.WSO2_JIRA_BASE_URL));
+        HashMap<String, String> ticketAssigneeMapper = new HashMap<>();
+        //assignees for each products
+        ticketAssigneeMapper.put(JSScannerConstants.AM, properties.getProperty(IssueCreatorConstants.APIM));
+        ticketAssigneeMapper.put(JSScannerConstants.IDENTITYSERVER, properties.getProperty(
+                IssueCreatorConstants.IDENTITYSERVER));
+        ticketAssigneeMapper.put(JSScannerConstants.INTEGRATION, properties.getProperty(IssueCreatorConstants
+                .INTEGRATION));
+        ticketAssigneeMapper.put(JSScannerConstants.STREAMPROCESSOR, properties.getProperty(IssueCreatorConstants
+                .STREAMPROCESSOR));
+        ticketAssigneeMapper.put(JSScannerConstants.OB, properties.getProperty(IssueCreatorConstants.OPENBANKING));
+        issueCreatorAPI.setAssigneeMapper(ticketAssigneeMapper);
```

Comment:
All of the above depends on property loading being successful. Therefore they should go inside the try block